### PR TITLE
chore: PR template to add reminder for checking actual mobile devices

### PR DESCRIPTION
**Pull request by sshanzel**
@rebelchris what do you think of this change? Do you think it can add value to our future PRs? If really necessary in some cases, we can probably suggest utilizing an emulator to further check.

With that, it might also be a good idea to create a doc page of what software we suggest for emulating different devices.

image

### CODE REVIEW

**Review by idoshamun — APPROVED** [review]
Great addition. But please wait for @rebelchris approval

**Review by rebelchris — APPROVED** [review]
Nice, great addition indeed 💖

**Pull request merged**
